### PR TITLE
hw 1

### PR DESCRIPTION
**Pull request by ilya449**

### CODE REVIEW

**Review by KostiantynVorobiov — APPROVED** [review]
Looks good!

File: src/main/java/com/test/sb/service/CsvFileReaderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
```
Comment (KseniiaMakarova) [inline src/main/java/com/test/sb/service/CsvFileReaderServiceImpl.java]:
```suggestion
public class FileReaderServiceImpl implements FileReaderService {
```
can it read only from csv files or from any files?

File: src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package com.test.sb.service;
+
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import static org.junit.Assert.assertEquals;
```
Comment (KseniiaMakarova) [inline src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java]: let's organize imports the right way here
you can configure the checkstyle plugin to check your tests

File: src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                    + "Lion, The Witch, and The Wardrobe\"\" - this is the treat that seduces "
+                    + "Edmund into selling out his Brother and Sisters to the Witch.\"");
+
+    @Autowired
+    private CsvFileReaderService service;
+
+    @Test
+    public void readFile_OK() {
+        List<String> actual = service.readFile(FILE_WITH_DATA_NAME);
+        actual.forEach(System.out::println);
```
Comment (KseniiaMakarova) [inline src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java]: do we need to sout the actual result in the final version?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java (hunk 2 above)
Comment (ilya449) [reply src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java]: How can I do it?

File: src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java (hunk 3 above)
Comment (ilya449) [reply src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java]: Forgot to delete it when tested, TY

File: src/main/java/com/test/sb/service/CsvFileReaderServiceImpl.java (hunk 1 above)
Comment (ilya449) [reply src/main/java/com/test/sb/service/CsvFileReaderServiceImpl.java]: Agree, not only from csv

File: src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java (hunk 2 above)
Comment (ilya449) [reply src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java]: I've found

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**